deployments/movement_ranges_maps.yaml

```diff
@@ -25,11 +27,14 @@ deployment:
   model:
     model_name: Central
     # model_name_description: TODO
-    release_type: One release
+    release_type: Many releases
```

Comment:
This is an interesting question.  It's a one time release, of many days of data.  One for future discussion.

Reply:
Filed: 
- #117